packages/app/src/app/pages/NewDashboard/Components/Selection/index.tsx

```diff
@@ -562,9 +570,50 @@ export const SelectionProvider: React.FC<SelectionProviderProps> = ({
     else if (new Date().getTime() - callbackCalledAt.current > 60) callback();
   };
 
-  const onContainerMouseUp = event => {
+  const onContainerMouseUp = () => {
     if (drawingRect) resetSelectionRect();
   };
+  if (page === 'repos') {
```

Comment:
Do we need a completely different one for the `'repos'` page?

Reply:
omg yes, there is noe fancy stuff, I don't think the contextMenu should be there tbh

Reply:
Do we still need it here in that case?

Reply:
spoke in slack